index.html

```diff
@@ -16,11 +16,42 @@
     <link rel="stylesheet" href="css/styles.css" />
   </head>
   <body>
-    <header>
-      <div class="container"></div>
-    </header>
+    <div class="container header">
+      <h1><strong>Code Journal</strong></h1>
+    </div>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div class="div" data-view="entry-form">
+        <form id="entry-form">
+          <div class="column-full">
+            <h1>New Entry</h1>
+          </div>
```

Comment:
`.column-full` won't have an impact unless it's inside a `row`. Since `column-full` has a `flex-basis` applied to it, it will only work if it's direct parent has `display: flex`. You need to wrap this inside a `div class="row"`